src/tsapp.h

```diff
@@ -117,6 +97,14 @@ struct TSApp : wxApp {
         return 0;
     }
 
+    void SetupLanguage() {
```

Comment:
Unless the code in question is crazy long & complicated, I generally feel that functions that are used exactly once make code readability worse, not better, compared to a block of code (with maybe a comment stating what the block does).